components/sections/home-page/Hero.tsx

```diff
@@ -34,10 +38,13 @@ const Hero = ({ data }: HeroProps) => {
               </Button>
             ))}
           </div>
+          <a className="text-sm text-wh pt-7" href="#">
+            Working with a team? Click here 
+          </a>
```

Comment:
This should be a link to the previous landing page at the `/teams` route.

Reply:
Do you mean https://opensauced.pizza/teams?

Reply:
> Do you mean https://opensauced.pizza/teams?

Correct